### PR TITLE
Create stackdriver plugin on request, and use a dedicated output for handling agents' logs

### PR DESCRIPTION
**Pull request by sophieyfang**
- Make a dedicated stackdirver output plugin for handling agents' log
- Aggregate logs handling by the exporter's ID mentioned in the pipeline.
   - if there are multiple receivers are going to the same exporter ID. Share the same exporter plugin among those logs stream by doing `Match_Regex ^(input-1|input-2)$`
- Create another testcase folder for having multiple stackdriver exporters

### CODE REVIEW

**Review by qingling128 — APPROVED** [review]
LGTM

**Review by qingling128 — COMMENTED** [review]
One thing worth checking is whether `Match ops-agent-fluent-bit,ops-agent-collectd` is supported by Fluent Bit. Or if we need to do:

```
Match_Regex ops-agent-fluent-bit|ops-agent-collectd
```

**Comment by sophieyfang** [comment]
> One thing worth checking is whether `Match ops-agent-fluent-bit,ops-agent-collectd` is supported by Fluent Bit. Or if we need to do:
> 
> ```
> Match_Regex ops-agent-fluent-bit|ops-agent-collectd
> ```

Right, I am modifying it to use Match_Regex as default setting. Thanks for catching that!

# Tested in centos8:
## Conf
```
[SERVICE]
    flush        5
    daemon       Off
    log_level    info
    parsers_file parsers.conf

    plugins_file plugins.conf
    http_server  Off
    http_listen  0.0.0.0
    http_port    2020

    storage.metrics on

[INPUT]
    Name        tail
    Path        /tmp/file1
    Tag 	file1

[INPUT]
    Name        tail
    Path        /tmp/file2
    Tag 	file2

[OUTPUT]
    name  stdout
    Match_Regex ^(file1|file2)$
```

## Verified

```
[sophieyfang_google_com@centos8-meoow ~]$ sudo /opt/td-agent-bit/bin/td-agent-bit -c /etc/td-agent-bit/td-agent-bit.conf --log_file=/etc/td-agent-bit/td-agent-bit.log &
[1] 44864
[sophieyfang_google_com@centos8-meoow ~]$ Fluent Bit v1.7.2
* Copyright (C) 2019-2021 The Fluent Bit Authors
* Copyright (C) 2015-2018 Treasure Data
* Fluent Bit is a CNCF sub-project under the umbrella of Fluentd
* https://fluentbit.io

[0] file1: [1615422998.102391854, {"log"=>"file 111111"}]
[0] file2: [1615423033.976546237, {"log"=>"file 2222"}]
```

**Review by qingling128 — APPROVED** [review]
LGTM

File: confgenerator/testdata/valid/default_config/golden_fluent_bit_main.conf [hunk 1]
```diff
@@ -148,23 +148,7 @@
...
-    tls         On
-    # Do not force certificate validation.
-    tls.verify  Off
-
-[OUTPUT]
-    # https://docs.fluentbit.io/manual/pipeline/outputs/stackdriver
-    Name     stackdriver
-    resource gce_instance
-    Match    syslog
+    Match_Regex    ^(syslog)$
```
Comment (qingling128) [inline confgenerator/testdata/valid/default_config/golden_fluent_bit_main.conf:151]: Can we align these fields? e.g.

```
    Name         stackdriver
    resource     gce_instance
    Match_Regex  ^(syslog)$
```

**Review by qingling128 — DISMISSED** [review]
Looks like Travis CI is failing.

File: confgenerator/testdata/valid/default_config/golden_fluent_bit_main.conf (hunk 1 above)
Comment (sophieyfang) [reply confgenerator/testdata/valid/default_config/golden_fluent_bit_main.conf:151]: > Looks like Travis CI is failing.

Need to update fluent/conf_test.go

Comment (sophieyfang) [reply confgenerator/testdata/valid/default_config/golden_fluent_bit_main.conf:151]: Fixed

**Comment by sophieyfang** [comment]
> Looks like Travis CI is failing.

File: confgenerator/testdata/valid/default_config/golden_fluent_bit_main.conf (hunk 1 above)
Comment (qingling128) [reply confgenerator/testdata/valid/default_config/golden_fluent_bit_main.conf:151]: [Format] Looks like the fields are still not aligned?

**Review by qingling128 — APPROVED** [review]
LGTM with a comment

**Pull request merged**